### PR TITLE
Dash Icon with cooldown UI

### PR DESCRIPTION
**Pull request by JianmingChen**

### CODE REVIEW

File: Assets/Scripts/Cooldown.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (mbajaman) [inline Assets/Scripts/Cooldown.cs:2]: Removed unused System using statement

File: Assets/Scripts/Cooldown.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Cooldown : MonoBehaviour
+{
+    [Header("DashBG")]
+    public Image dashCooldown;
+    private float cooldown = 3;
```
Comment (mbajaman) [inline Assets/Scripts/Cooldown.cs:10]: Private variable should be named with underscore: _cooldown

File: Assets/Scripts/Cooldown.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Cooldown : MonoBehaviour
+{
+    [Header("DashBG")]
+    public Image dashCooldown;
```
Comment (mbajaman) [inline Assets/Scripts/Cooldown.cs:9]: Public variables should be PascalCase = DashCooldown

File: Assets/Scripts/Cooldown.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    private float cooldown = 3;
+    bool isCooldown = false;
+    public KeyCode dashCode;
```
Comment (mbajaman) [inline Assets/Scripts/Cooldown.cs:12]: Public variable should be PascalCase

File: Assets/Scripts/Pause.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+public class Pause : MonoBehaviour
+{
+    [SerializeField] GameObject pauseMenu;
+
+    public void pause()
```
Comment (AlexisCM) [inline Assets/Scripts/Pause.cs:11]: Classes should be PascalCase

File: Assets/Scripts/Pause.cs [hunk 6]
```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+public class Pause : MonoBehaviour
+{
```
Comment (AlexisCM) [inline Assets/Scripts/Pause.cs:8]: No line break before opening curly brace

File: Assets/Scripts/Pause.cs [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    [SerializeField] GameObject pauseMenu;
+
+    public void pause()
+    {
+        pauseMenu.SetActive(true);
+        Time.timeScale = 0f; //freeze the game
+    }
+
+    public void Resume()
+    {
```
Comment (AlexisCM) [inline Assets/Scripts/Pause.cs:18]: No line break before opening curly brace

File: Assets/Scripts/Pause.cs [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        pauseMenu.SetActive(false);
+        Time.timeScale = 1f; //active the game
+    }
+
+    public void home(int sceneID) {
```
Comment (AlexisCM) [inline Assets/Scripts/Pause.cs:23]: Classes should be PascalCase

**Review by AlexisCM — CHANGES_REQUESTED** [review]

**Pull request closed without merging**